var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -70,11 +70,15 @@ def __build_mods_coreneuron(self, mods_location, link_flag, include_flag):
         spec = self.spec
         assert os.path.isdir(mods_location), mods_location
         include_flag += ' -I%s' % (spec['coreneuron'].prefix.include)
-        which('nrnivmodl-core')(
-            '-i', include_flag, '-l', link_flag, '-n', self.mech_name,
-            '-v', str(spec.version), '-c', mods_location)
+        if spec.satisfies('^coreneuron@develop') or spec.satisfies('^coreneuron@master'):
```

Comment:
@iomaganaris : I think we should do opposite logic here i.e. if version is <= 0.16 (latest release) then use old logic else do new options. This way when there will be new release, it will work as well.